src/tools/pmempool/common.h

```diff
@@ -72,10 +72,10 @@
 #define	min(a, b) ((a) < (b) ? (a) : (b))
 #define	ENTIRE_UINT64 (\
 {\
-struct range ret = {\
+struct range __entire_uint64 = {\
```

Comment:
I don't like this double underscore, as this is by convention the prefix reserved for the compiler internal use.